localstack/testing/snapshots/transformer_utility.py

```diff
@@ -483,6 +483,8 @@ def cloudwatch_api():
         """
         return [
             TransformerUtility.key_value("AlarmName"),
+            TransformerUtility.key_value("MetricName"),
```

Comment:
NIT: i think using a random `Namespace` should be sufficient. The `MetricName` could stay the same for most test cases.

Reply:
Yeah was thinking the same but only after I finished it :grimacing: 
Will adjust